src/main/java/org/apache/maven/report/projectinfo/CiManagementReport.java

```diff
@@ -225,23 +199,29 @@ else if ( isCiManagementSystem( system, "travis" ) )
         }
 
         /**
-         * Checks if a CI management system is bugzilla, continuum...
+         * Search system description.
          *
-         * @return true if the CI management system is bugzilla, continuum..., false otherwise.
+         * @param system a system for description
+         * @return system description from properties
          */
-        private boolean isCiManagementSystem( String system, String actual )
+        private String getIntroForManagementSystem( String system )
```

Comment:
What management system? The "Ci" is gone.

Reply:
method isCiManagementSystem was used in many if .. else ...
Now we check list/set of prefix for CI system

Reply:
What I meant is that the term "Ci" is gone: `getIntroForCiManagementSystem()` instead of `getIntroForManagementSystem()`.

Reply:
method renamed